ocaml/ocamltest/result.ml

```diff
@@ -15,7 +15,7 @@
 
 (* Definition of test-result related types and functions *)
 
-type status = Pass | Skip | Fail
+type status = Pass | Skip | Fail | Predicate of bool
```

Comment:
I wonder whether the new cases
could/should be nested under
existing constructors. The `is_pass`
and `is_skip` functions seem to
suggest predicates are ways of
passing / skipping.
That being said, we probably
don't want to spend time on
such questions.

Reply:
Not a bad idea indeed, but I've just left a CR for the moment.

Reply:
(We shouldn't spend too long on this as it needs to go upstream, and there may be further discussion there resulting in changes.)